source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,39 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-06-30
+
+### Highlights
+
+### Unlimited E10/20 trials
```

Comment:
```suggestion
### Unlimited E10/20 trials prohibited
```

Reply:
1. I would remove from Cloud changelog, since it only affects Self-Managed (Cloud has no E10/E20 trials).
2. For self-managed changelog on 7/16 release, I would adjust this to say

```
#### Improved Enterprise trial experience

- After a Self-Managed trial ends, admins can optionally contact sales or make a purchase in a single click.
```